test/functional/ios/safari/windows-frame-specs.js

```diff
@@ -50,9 +50,12 @@ describe('safari - windows and frames (' + env.DEVICE + ') @skip-ios6"', functio
         .elementById('only_on_page_2')
         .nodeify(done);
     });
-    it("should be able to open js popup windows with safariAllowPopups set to true", function (done) {
+
+    // broken on real devices, see https://github.com/appium/appium/issues/5167
+    it("should be able to open js popup windows with safariAllowPopups set to true @skip-real-device", function (done) {
```

Comment:
yeah, not surprising. any capability that works by modifying simulator plist files isn't going to be supported on real devices. we probably need a doc about this (or to put "simulator-only" in caps.md)